scripts/develop-module.js

```diff
@@ -0,0 +1,299 @@
+const fs = require('fs');
```

Comment:
Should add `#!/usr/bin/env node` to the cop and make sure this has execute permissions, so we can run it directly without `node`